db/organizations/sitecore.eno

```diff
@@ -0,0 +1,6 @@
+name: Sitecore
+website_url: https://www.sitecore.com/
+privacy_policy_url: https://www.sitecore.com/legal/privacy-policy
+privacy_contact: privacy@sitecore.com
+country: DK
```

Comment:
Looks correct. The English Wikipedia says it is now US headquartered, but I did not find any sources.

Both Danish and German Wikipedia list it as headquartered in Denmark as well.

Reply:
I should have clarified: https://www.sitecore.com/legal/privacy-policy#:~:text=SECTION%201.%20WHO%20WE%20ARE Lists the DK company number. 

FYI, in general if I do not specify a reference for the country, it is because it is shown in the privacy policy.

Reply:
All good. I only mentioned it here, because it is a very rare case where it conflicted with the English Wikipedia. That confused me a bit.

Thanks for the reference!